modules/local/interpro_list_getdb.nf

```diff
@@ -7,21 +7,21 @@ process INTEPRO_ENTRY_LIST_GETDB {
     publishDir "${params.dbs}", mode: 'copy'
 
     output:
-    tuple path("ipr-entry-lists/", type: "dir"), val("94.0"), emit: interpro_entry_list
+    tuple path("interproscan_entry_list/", type: "dir"), val("94.0"), emit: interpro_entry_list
```

Comment:
This one (and in all other occurrences) should be interpro entry list, not interproscan